server/supernova/audio_backend/portaudio_backend.hpp

```diff
@@ -101,8 +101,16 @@ class portaudio_backend:
         for (int i = 0; i != device_number; ++i) {
             const PaDeviceInfo * device_info = Pa_GetDeviceInfo(i);
             if (device_info) {
+                #ifndef __APPLE__
+                //list APIs on Windows
+                const PaHostApiInfo * api_info = Pa_GetHostApiInfo(device_info->hostApi);
+                printf("%d: %s : %s (%d inputs, %d outputs)\n", i, api_info->name,
+                       device_info->name, device_info->maxInputChannels,
+                       device_info->maxOutputChannels);
+                #else
                 printf("%d: %s (%d inputs, %d outputs)\n", i, device_info->name,
                        device_info->maxInputChannels, device_info->maxOutputChannels);
+                #endif
```

Comment:
`#ifndef __APPLE__` -- why not `#ifdef _WIN32` as you wrote in SC_PortAudio.cpp above? also, left-align both sets of preprocessor directives in this file